admin/resources/css/app.css

```diff
@@ -19,3 +19,32 @@ table tbody tr:nth-of-type(odd) {
 table tbody tr:nth-of-type(even) {
     background-color: #FAFAFA;
 }
+
+.reset-ul {
+    list-style: none;
+    padding: 0;
+}
+
+body {
+    position: absolute;
+    margin: 0px;
+    overflow: hidden;
+    top: 0px;
+    bottom: 0px;
+    width: 100%;
+}
+
+.container {
+    height: 100vh;
+    margin: 0;
+    display: flex;
+    flex-direction: column;
+}
```

Comment:
H2 has a container attribute. Its different from this one? https://hydrogen.design/#containers

Reply:
Yea this is different. I used it to stack the content and footer correctly. Also, it should be easy to add the goc banner and header as well with this setup.